clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ResolvedChain.java

```diff
@@ -73,7 +79,11 @@ public Chain getCompactedChain() {
 
     @Override
     public boolean isCompacted() {
-      return compacted;
+      return compactedSize > 0;
+    }
+
+    public int getCompactedSize() {
```

Comment:
this getter reads like it is the compacted chain size, but the use shows it is something different.
What about `getCompactionCount` or something else?